README.md

```diff
@@ -55,179 +22,53 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Hello_Microverse] <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
```

Comment:
Please remove this text since it is an instruction for you to create a proper README. It is important to have a good README file to have a professional repository.  👍